images/warp10/s6/cont-init.d/10-generate-hash-seeds

```diff
@@ -8,4 +8,5 @@ WARP10_SECRETS="$WARP10_CONFIG_DIR/00-secrets.conf"
 if [ ! -f "$WARP10_SECRETS" ]; then
     cp "$WARP10_CONF_TEMPLATES/00-secrets.conf.template" "$WARP10_SECRETS"
     python "${WARP10_HOME}/etc/generate_crypto_key.py" "$WARP10_SECRETS"
+    echo "warp10.manager.secret = scality" >> $WARP10_SECRETS
```

Comment:
Why not calling `generate_crypto_key.py` here ?

Reply:
Internally `generate_crypto_key.py` is using a regex to insert random values for the secrets.
There's 2 problems with this:
- we need the management secret to be known to the hard limit process and random keys makes this difficult at deploy
- The line for `warp10.manager.secret` doesn't contain the template value `generate_crypto_key.py` uses to insert

Reply:
ok, we might solve this later, plz create a ticket.